examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/OfferingFragment.kt

```diff
@@ -70,10 +59,6 @@ class OfferingFragment : Fragment(), PackageCardAdapter.PackageCardAdapterListen
     }
 
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
-        Purchases.sharedInstance.getCustomerInfoWith {
-            activeSubscriptions = it.activeSubscriptions
-        }
```

Comment:
Hmm can you explain this? Maybe it's not needed... but not sure.

Reply:
whoops! I had done his initially because I was going to move everything to a viewModel, then decided it was out of scope for the PR, and I guess I forgot to revert these lines, thanks for bringing it up!